README.md

```diff
@@ -235,58 +235,75 @@ For headaches, take Ibuprofen
-module's go.mod.
+The version "latest" matches the latest available tagged version for the module
+containing the main package. gobin will make a network request to resolve
+"latest" unless -nonet is provided, in which case it will attempt to resolve
+"latest" within the module download cache. Versions that take the form of a
+revision identifier (a branch name, for example) can only be resolved with a
+network request and hence are incompatible with -nonet.
+
+If no version is specified for a main package, gobin behaves differently
+depending on whether the -m flag has been provided.  By default, gobin attempts
+to resolve the latest tagged version available in the module download cache (or
+via the network if the -u flag is provided). If the -m flag is provided, gobin
+attempts to resolve the current version via the main module's go.mod. If this
+resolution fails in either case, "latest" is assumed as the version.
 
 By default, gobin installs the main packages to $GOBIN (or $GOPATH/bin if GOBIN
 is not set, which defaults to $HOME/go/bin if GOPATH is not set).
 
-The -r flag takes exactly one main package argument and runs that package.  It
-is similar therefore to go run. Any arguments after the single main package
+The -run flag takes exactly one main package argument and runs that package.
+It is similar therefore to go run. Any arguments after the single main package
 will be passed to the main package as command line arguments.
 
-The -p flag prints the cache install path for each of the provided packages
-once versions have been resolved.
-
-The -r and -p flags are mutually exclusive.
+The -p flag prints the gobin install cache path for each of the provided
+packages once versions have been resolved.
 
-The -g flag forces gobin to perform a network fetch for the provided main
+The -d flag instructs gobin to stop after installing the packages to the gobin
+install cache; that is, it instructs gobin not to install, run or print the
 packages.
 
-The -n flag prevents network access and instead uses the GOPATH module download
-cache where required.
+The -run, -p and -d flags are mutually exclusive.
+
+The -u flag forces gobin to check for the latest tagged version. As such, it
+has no effect if any form of version is specified.
 
-The -g and -n flags are mutually exclusive.
+The -nonet flag prevents network access and instead uses the module download
+cache where required for version resolution, including "latest".
 
-The -m flag causes gobin to use the main module (the module containing the
-directory where the gobin command is run). The main module is given by go env
-GOMOD. Without this flag gobin effectively runs as a "global" tool.
+The -u and -nonet flags are mutually exclusive.
 
 It is an error for a non-main package to be provided as a package argument.
 
+
+Cache directories
+=================
+
+By default, gobin installs the main packages to the directories
```

Comment:
I see what you're getting at. But I don't think the suggested wording works. Because `gobin` _always_ installs a copy to its cache. That cache location is a function of the presence of `-m`. It only creates a "copy" in case "no flag" (awkward) is provided, i.e. it's not `-run`, `-p` or `-d`.

Rewording per our conversation on Slack